egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -16,8 +16,12 @@ decode_dir_affix=
 num_epochs=4
 remove_egs=false
 common_egs_dir=
-
 affix=
+
+# decoding options
+extra_left_context=50
```

Comment:
You should probably keep these. I would not recommend using variables for
things like --initial-learning-rate,..
but use variables for any value which is required more than once in the
script.

On Wed, Apr 20, 2016 at 9:28 PM, Xingyu Na <notifications@github.com> wrote:

> In egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r60514554>:
>
> >  affix=
> > +
> > +# decoding options
> > +extra_left_context=50
>
> Problem is that there are no $chunk_{left,right}_context variables
> anymore. I remove the options and use the value 40 as Vimal suggested. 在
> 2016/4/21 5:23, Daniel Povey 写道:
> … <#m_8923994972546964763_m_7854474476791396357_m_-7028054375423755619_>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/657/files/f9483fdcb59dbfa711df31c41c2bea90b23c142c..ea76204bc64d9a139597e488a7821d5e44dc3bd4#r60514554>
>